### DISCUSSION TITLE
Repeating message caskroom/cast was moved

### DISCUSSION THREAD

**Discussion by bcodermicro** [Homebrew/discussions#2786] [Everyday usage]
`user@server sites-available % brew update
fatal: could not read Username for 'https://github.com': terminal prompts disabled
Error: homebrew/homebrew-science does not exist! Run `brew untap homebrew/homebrew-science` to remove it.
Updated 1 tap (homebrew/core).
==> Updated Formulae
awscli                                   ctlptl                                   rqlite                                   vite
Error: caskroom/cask was moved. Tap homebrew/cask instead.
user@server sites-available % brew tap
homebrew/cask
homebrew/cask-fonts
homebrew/core
homebrew/nginx
homebrew/science
homebrew/services
`

**Comment by gromgit** [comment]
If by:

> Repeating message caskroom/cast was moved

you actually mean:

> `Error: caskroom/cask was moved. Tap homebrew/cask instead.` keeps appearing with every `brew` command I run

then you need to do the following:
```
brew tap homebrew/cask
brew untap caskroom/cask
```

If that second command fails, then `rm -fr $(brew --repo caskroom/cask)` instead.
